imessage-exporter/src/app/options.rs

```diff
@@ -108,6 +116,18 @@ impl<'a> Options<'a> {
             )));
         }
 
+        // Ensure that if iOS is enabled, that the db_path is to a backup
+        if ios && user_path.is_some() {
+            let db_path = PathBuf::from(user_path.unwrap());
+            if !db_path.join(DEFAULT_IOS_CHATDB_PATH).exists() {
+                return Err(RuntimeError::InvalidOptions(format!(
+                    "Option {OPTION_IOS} is enabled, but the database path does not appear to be a valid iOS backup"
```

Comment:
This message should accurately reflect what went wrong here: it's not invalid, the file was not found at the path specified.